windows/bleachbit.nsi

```diff
@@ -226,7 +226,6 @@ VIFileVersion ${File_VERSION}
 ; MUI_UNPAGE_DIRECTORY not needed, ATM.
 ; !insertmacro MUI_UNPAGE_DIRECTORY
 !insertmacro MUI_UNPAGE_COMPONENTS
-UninstallText $(BLEACHBIT_UNINSTALL_TEXT)
```

Comment:
Why this line is removed?

Reply:
This line is duplicated in the installer on line #427.